packages/app-builder-lib/src/targets/nsis/nsisUtil.ts

```diff
@@ -38,6 +38,19 @@ export const NSIS_PATH = () => {
   })
 }
 
+export const NSIS_RESOURCES_PATH = () => {
+  return NsisTargetOptions.then((options: NsisOptions) => {
+    if (options.customNsisResources) {
+      const { checksum, url, version } = options.customNsisResources
+      if (checksum && url) {
+        const binaryVersion = version || checksum.substr(0, 8)
```

Comment:
Wait, what is happening here?

Reply:
No idea, I just copy-pasted it from above 😂 I don't think we actually need it?

Reply:
Fixed.